test/models/M3D/README.md

```diff
@@ -0,0 +1,14 @@
+Model 3D Samples
+================
+
+ aliveai_character.m3d - from Minetest aliveai mod (textures, animations, original 47k, m3d 2.5k)
```

Comment:
License?

Reply:
They are from Open Source games. Should I only keep the models converted from Assimp samples? I can do that.